lib/standard-eslint-ts.js

```diff
@@ -5,6 +5,14 @@
 const { rules } = require('@typescript-eslint/eslint-plugin')
 const standard = require('eslint-config-standard')
 
+function deepCopyRuleValue (ruleValue) {
+  if (!Array.isArray(ruleValue)) {
+    // Case: rule value is a primitive (e.g., `1`, `"off"`, `"error"`)
+    return ruleValue
+  }
+  return JSON.parse(JSON.stringify(ruleValue))
```

Comment:
why not `structuredClone`?

Reply:
Thanks for the suggestion! Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone), `structuredClone` is supported starting Node.js 17, which doesn't match what we have in `package.json` right now, which is 

```json
    "node": "^16.10.0 || >=18.0.0"
```

For backwards compatibility, I just pushed https://github.com/SocketDev/eslint-config/pull/69/commits/0a04417abd049515c2aead0e72eb854c1dc6c815 which uses structuredClone if available and falls back to JSON serialization and deserialization otherwise. Does this seem fine?